js/libs/ui-shared/src/main.ts

```diff
@@ -1,5 +1,6 @@
 export { ContinueCancelModal } from "./continue-cancel/ContinueCancelModal";
 export { SelectControl } from "./controls/SelectControl";
+export type { Option } from "./controls/SelectControl";
```

Comment:
If we're going to be exporting this type, I think we should rename it so it better matches the control it's associated with.
```suggestion
export type { SelectOption } from "./controls/SelectControl";
```

Reply:
That would name clash with SelectOption of patternfly

Reply:
Then perhaps `SelectControlOption`?